README.md

```diff
@@ -1,21 +1,24 @@
 # perf-triage
 Tools used for triaging performance bugs.
 
-### send-reminder.py
-Note: this script is a WIP and we're working on integrating it with the other
-scripts.
+## Setup
 
-The first time you run, you'll need to create a virtualenv and install the
-dependencies:
-```sh
-# Create virtualenv
-python3 -m venv venv
+For convenience, you can use poetry to manage dependencies and virtual environments.
 
-# Activate virtualenv
-source venv/bin/activate
+When running for the first time, you will need to [install poetry](https://python-poetry.org/docs/#installation) and then run `poetry install` to create the virtual environment and install dependencies.
```

Comment:
What are do we get out of poetry that we don't get out of venv? I haven't used it before. If we don't have a strong reason to use it over venv, I'd prefer to use venv to reduce our external dependencies, reducing our attack surface area (the target being our dev machines or CI systems).

Reply:
Mostly just convenience. If you'd prefer to continue using venv directly that works for me. We can close this pull request.

Reply:
Up to you! I don't know poetry and who maintains it so I don't have any context to trust it.

Reply:
I'll close this for now, we can always revisit if it feels valuable in the future.